annotated-projects/PXD000547/PXD000547.idf.tsv

```diff
@@ -0,0 +1,34 @@
+MAGE-TAB Version	1.1
+Investigation Title	Proteome of the Human Corpus Callosum
+Experiment Description	Characterization of the human Anterior Temporal Lobe and Corpus Callosum: C-HPP
+
+Date of Experiment	2014-06-05
+Public Release Date	2014-06-05
+
+Protocol Name	P-MTAB-Sample-PXD000547	P-MTAB-Data-PXD000547
+Protocol Type	sample collection protocol	data analysis protocol
+Protocol Description	Protein samples extracted from human brains in 7 M urea, 2 M thiourea,4% CHAPS, 2% ASB-14, and 70 mM DTT. Proteins were pre-fractionated by SDS-PAGE and submitted to in-gel digestion. Peptides were separated in a nano HPLC, coupled to an LTQ-Orbitrap XL mass spectrometer.	Mass spectra were extracted in the MS2 format using the PatternLab for Proteomics RawReader employing the NeXtProt database.
+Protocol Parameters
+Protocol Hardware	LTQ Orbitrap
+Protocol Software
+Protocol Contact
+
+Person Last Name	Martins-de-Souza	Martins-de-Souza
```

Comment:
Is it usual in `IDF` to have this repetition?

Reply:
I guess the person there https://www.ebi.ac.uk/pride/archive/projects/PXD000547 is the submitter and the lab head, has two roles. I need to update the script that generates the IDF to add two roles rather that have two persons, BTW, the user added two affiliation 🧐

Reply:
Yes, I noticed that it happens if the submitter is also the PI. So it will be an easy fix then:)

Reply:
Yes, in case of >1 role, you can add the role terms in one field separated with semicolons.